src/components/Card/__tests__/Card.test.tsx

```diff
@@ -11,39 +11,29 @@ jest.mock('react-router-dom', () => ({
 }));
 
 describe('Card', () => {
-    it('should render card with single column', () => {
+    it('should render card with team data', () => {
         const id = 'test-id';
-        const columns = [{key: 'columnKey', value: 'columnValue'}];
+        const columns = [{key: 'name', value: 'columnValue'}];
         render(<Card id={id} columns={columns} />);
 
-        expect(screen.queryAllByTestId('card-column')).toHaveLength(columns.length);
 
-        expect(screen.getByText('columnKey')).toBeInTheDocument();
-        expect(screen.getByText('columnValue')).toBeInTheDocument();
+        expect(screen.getByTestId('card-name')).toBeInTheDocument();
     });
 
-    it('should render card with multiple columns', () => {
+    it('should render card with user data', () => {
```

Comment:
Updated tests for the component, to test for Team data, User Data, and the navigations tests